code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -97,6 +97,8 @@
 		if("hiss", "hisses")
 			if(isunathi(src)) //Only Unathi can hiss.
 				on_CD = handle_emote_CD()			//proc located in code\modules\mob\emote.dm'
+			if(isvox(src)) //Only Vox can hiss.
```

Comment:
Best to just include this in the check on line 98. Also the comments are conflicting!!!